src/component/strategies/StrategiesList/StrategiesList.tsx

```diff
@@ -251,56 +335,86 @@ export const StrategiesList = () => {
         />
     );
 
-    const strategyList = () =>
-        strategies.map(strategy => (
-            <ListItem key={strategy.name} className={styles.listItem}>
-                <ListItemAvatar>
-                    <Extension color="disabled" />
-                </ListItemAvatar>
-                <ListItemText
-                    primary={strategyLink(strategy?.name, strategy?.deprecated)}
-                    secondary={strategy.description}
-                />
-                <ConditionallyRender
-                    condition={strategy.deprecated}
-                    show={reactivateButton(strategy)}
-                    elseShow={deprecateButton(strategy)}
-                />
-                <ConditionallyRender
-                    condition={hasAccess(UPDATE_STRATEGY)}
-                    show={editButton(strategy)}
-                />
-                <ConditionallyRender
-                    condition={hasAccess(DELETE_STRATEGY)}
-                    show={deleteButton(strategy)}
-                />
-            </ListItem>
-        ));
-
     const onDialogConfirm = () => {
         dialogueMetaData?.onConfirm();
-        setDialogueMetaData(prev => ({ ...prev, show: false }));
+        setDialogueMetaData((prev: IDialogueMetaData) => ({
+            ...prev,
+            show: false,
+        }));
     };
 
     return (
         <PageContent
-            header={<PageHeader title="Strategies" actions={headerButton()} />}
+            header={
+                <PageHeader
+                    title="Strategies"
+                    actions={
+                        <>
+                            <TableSearch
+                                initialValue={globalFilter}
+                                onChange={setGlobalFilter}
+                            />
+                            <PageHeader.Divider />
+                            <AddStrategyButton />
+                        </>
+                    }
+                />
+            }
         >
-            <Dialogue
-                open={dialogueMetaData.show}
-                onClick={onDialogConfirm}
-                title={dialogueMetaData?.title}
-                onClose={() =>
-                    setDialogueMetaData(prev => ({ ...prev, show: false }))
-                }
-            />
-            <List>
+            <div ref={ref}>
```

Comment:
Could pass `isLoading` to `PageContent` instead of using `useLoading` in this file.

Reply:
Thanks. Will do.